src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ProjectDebugPropertyPage.xaml

```diff
@@ -66,7 +66,13 @@
 
   <BoolProperty Name="HotReloadEnabled"
                 DisplayName="Enable Hot Reload"
-                Description="Apply code changes to the running application." />
+                Description="Apply code changes to the running application.">
+    <BoolProperty.Metadata>
+      <NameValuePair Name="VisibilityCondition">
+        <NameValuePair.Value>(has-project-capability "SupportsHotReload")</NameValuePair.Value>
```

Comment:
Oh yeah, I forgot about this one. Glad this was even more straightforward than expected.